src/odemis/driver/test/tescan_test.py

```diff
@@ -575,7 +584,7 @@ def setUp(self):
 
         # reset resolution and dwellTime
         self.scanner.scale.value = (1, 1)
-        self.scanner.resolution.value = (512, 256)
+        self.scanner.resolution._value = (512, 256)
```

Comment:
You should never touch a VA this way in "user" code. Why do you have to do that?